Rudder-Facebook/Classes/RudderFacebookIntegration.h

```diff
@@ -13,7 +13,10 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface RudderFacebookIntegration : NSObject<RSIntegration>
+@interface RudderFacebookIntegration : NSObject<RSIntegration> {
+    NSArray* events;
+    NSArray *TRACK_RESERVED_KEYWORDS;
```

Comment:
Variable names should not be in uppercase; they should follow camelCase convention.